src/signatures.jl

```diff
@@ -262,7 +274,7 @@ function set_to_running_name!(@nospecialize(recurse), replacements, frame, metho
     replacements[callee] = cname
     mi = methodinfos[cname] = methodinfos[callee]
     src = frame.framecode.src
-    replacename!(src.code[mi.start:mi.stop], callee=>cname)        # the method itself
+    replacename!(@view(src.code[mi.start:mi.stop]), callee=>cname) # the method itself
```

Comment:
This needs to be `@view` since otherwise we will miss to rename a simple `Symbol` statement at L396.